src/NServiceBus.RavenDB/Outbox/RavenDbOutboxStorage.cs

```diff
@@ -37,7 +37,8 @@ protected override void Setup(FeatureConfigurationContext context)
                 .ConfigureProperty(c => c.StoreToInstall, store);
 
             context.Container.ConfigureComponent<OutboxPersister>(DependencyLifecycle.InstancePerCall)
-                .ConfigureProperty(x => x.DocumentStore, store);
+                .ConfigureProperty(x => x.DocumentStore, store)
+                .ConfigureProperty(x => x.EndpointName, context.Settings.EndpointName());
```

Comment:
If this doesnt need anything else injected i would prefer we started favouring the delegated constructor api instead of the reflected property api